scripts/genunicodetables.lua

```diff
@@ -17,18 +17,22 @@
 --    which don't have ambiguous or double width, and emoji_all has all Emojis.
 if arg[1] == '--help' then
   print('Usage:')
-  print('  genunicodetables.lua UnicodeData.txt CaseFolding.txt ' ..
-        'EastAsianWidth.txt emoji-data.txt')
-  print('                       unicode_tables.generated.h')
+  print('  genunicodetables.lua unicode/ unicode_tables.generated.h')
   os.exit(0)
 end
 
-local unicodedata_fname = arg[1]
-local casefolding_fname = arg[2]
-local eastasianwidth_fname = arg[3]
-local emoji_fname = arg[4]
+local basedir = arg[1]:gsub('[/\\]$', '')
```

Comment:
I wasn't sure how well it would be handled on Windows (and don't have access to try it).  I figured it was safest to strip it.